src/com/xilinx/rapidwright/util/FileTools.java

```diff
@@ -175,40 +177,174 @@ public class FileTools {
         try {
-            fos = new FileOutputStream(fileName);
+            return getZstdOutputStream(new FileOutputStream(fileName));
+        } catch (IOException e) {
+            throw new UncheckedIOException(e);
+        }
+    }
+
+    /**
+     * Wraps the provided output stream with a Zstandard compression stream.
+     * 
+     * @param os The existing output stream.
+     * @return The new output stream that will use Zstandard compression.
+     */
+    public static OutputStream getZstdOutputStream(OutputStream os) {
+        try {
+            return new ZstdOutputStream(os, Params.RW_ZSTD_COMPRESSION_LEVEL);
+        } catch (IOException e) {
+            throw new UncheckedIOException(e);
+        }
+    }
+
+    /**
+     * Creates a Kryo output stream that instantiates a gzip compression stream to
+     * an output file.
+     * 
+     * @param fileName Name of the file to target.
+     * @return The created kryo-gzip output file stream.
+     */
+    public static Output getKryoGzipOutputStream(String fileName) {
+        try {
+            return getKryoGzipOutputStream(new FileOutputStream(fileName));
         } catch (FileNotFoundException e) {
-            e.printStackTrace();
+            throw new UncheckedIOException(e);
         }
-        return getKryoOutputStream(fos);
     }
 
-    public static Output getKryoOutputStream(OutputStream os) {
+    /**
+     * Creates a Kryo output stream that instantiates a gzip compression stream to
+     * an output stream.
+     * 
+     * @param fileName Name of the file to target.
+     * @return The created kryo-gzip output file stream.
+     */
+    public static Output getKryoGzipOutputStream(OutputStream os) {
         return getKryoOutputStreamWithoutDeflater(new DeflaterOutputStream(os));
     }
 
+    /**
+     * Use {@link #getKryoGzipOutputStream(String)} instead.
+     * @deprecated To be removed in 2023.2.0
+     */
+    public static Output getKryoOutputStream(String fileName) {
+        return getKryoGzipOutputStream(fileName);
+    }
+
+
+    /**
+     * Use {@link #getKryoGzipOutputStream(OutputStream)} instead.
+     * @deprecated To be removed in 2023.2.0
+     */
+    public static Output getKryoOutputStream(OutputStream os) {
+        return getKryoGzipOutputStream(os);
+    }
+
+    /**
+     * Wraps the provided output stream with a kryo stream. Will call
+     * {@link #useUnsafeStreams()} to decide on using unsafe or not.
+     * 
+     * @param os The output stream to wrap.
+     * @return The created kryo stream.
+     */
     public static Output getKryoOutputStreamWithoutDeflater(OutputStream os) {
         return useUnsafeStreams() ? new UnsafeOutput(os)
                                   : new Output(os);
     }
 
+    /**
+     * Creates a Kryo input stream that instantiates a Zstandard compression stream
+     * to an input file.
+     * 
+     * @param fileName Name of the file to target.
```

Comment:
```suggestion
     * Creates a Kryo input stream from decompressing a Zstandard compressed input
     * file
     * 
     * @param fileName Name of the file to read from.
```
I think many of the "input" variants below this one could also be reworded to be clearer?